app/views/works/index.html.erb

```diff
@@ -0,0 +1,51 @@
+<h1>List of Works</h1>
+ 
+<div>
+<h2>Albums</h2>
+<ul>
+<% Work.sort_albums.each do |album| %>
```

Comment:
Note that these 3 blocks of code for each category are nearly the same. Consider how you could DRY this up with a partial view or loop.